### ISSUE TITLE
Everything broken

### ISSUE THREAD

**Issue by PradyunS** [wasdennnoch/AndroidN-ify#1221]
All the lockscreen text is gone and the notification area is nonexistent

Module version 778
[xposed_error_20161116_110113.txt](https://github.com/wasdennnoch/AndroidN-ify/files/595690/xposed_error_20161116_110113.txt)

**Comment by wasdennnoch** [comment]
Are you using stock CM?

**Comment by PradyunS** [comment]
Yeah official cm 13

**Comment by yzyboost** [comment]
[xposed_error_20161118_193749.txt](https://github.com/wasdennnoch/AndroidN-ify/files/599837/xposed_error_20161118_193749.txt)

Same bug. Here's my log. Using aicp rom 6.0.1. 

Build 774-783 (no lockscreen text & notification drawer)

**Comment by yzyboost** [comment]
![screenshot_20161118-194421](https://cloud.githubusercontent.com/assets/23522663/20429088/76cf7528-adc7-11e6-81ca-fe8288d2e5cf.png)

**Comment by yzyboost** [comment]
![screenshot_20161118-194412](https://cloud.githubusercontent.com/assets/23522663/20429117/94eab608-adc7-11e6-8618-9dd4c5ff06d5.png)

**Comment by yzyboost** [comment]
Any fix for that issue?

**Comment by yzyboost** [comment]
Build 779 logcat. ( Updating build 772 into 783, no lockscreen (clock & date) & statusbar
xposed_error_20161118_065515.txt

As of now, I'm using build 783, same problem. All i can see is my wallpaper, no lockscreen & statusbar. When I deactivated the "fix huge...." in experimental the lockscreen & statusbar back. I want the build 783 without deactivating the fix huge under the experimental setting. Thanks! Cheers.

**Comment by paphonbth** [comment]
From provided logs, it seems like Xposed can't find QSPanel.setTiles which is normal on CM ROM as it has QSDragPanel instead. But the weird thing is that if QSDragPanel exists, it should hook that class instead of QSPanel.
I can't find the cause of this yet. This is very weird.

**Comment by yzyboost** [comment]
Ohmy!! I know you can fix that issue. Hehe. Cheers!

**Comment by kurtextrem** [comment]
I guess a bisect between 762 and 783 would be a start point. 762 is working without problems.

**Comment by PradyunS** [comment]
Im on 771 and that i think is the last working version

**Comment by yzyboost** [comment]
Is this fix?

**Comment by yzyboost** [comment]
Any update for this issue? :/

**Comment by yzyboost** [comment]
Update?

**Comment by PradyunS** [comment]
I thought on 785 it would be fixed because the description is "Fixed crash on loading tiles (#1239) but nah still same problem

**Comment by jackbuehner** [comment]
Try turning off "Fix huge space under qs header" in Experimental settings.

**Comment by yzyboost** [comment]
Yeah we know that. But we need an update that will not turning off the fix
huge....

On Nov 29, 2016 8:54 PM, "Jack Buehner" <notifications@github.com> wrote:

Try turning off "Fix huge space under qs header" in Experimental settings.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/wasdennnoch/AndroidN-ify/issues/1221#issuecomment-263562845>,
or mute the thread

**Comment by yzyboost** [comment]
[Logcat1.txt](https://github.com/wasdennnoch/AndroidN-ify/files/620977/Logcat1.txt)
@paphonbth

**Comment by yzyboost** [comment]
Any update? @paphonbth @wasdennnoch

**Comment by PradyunS** [comment]
You don't have to ask every single day when an update comes out, try it, if our worried it works if it doesn't oh well

**Comment by PradyunS** [comment]
*if it works

**Comment by yzyboost** [comment]
Still no update for this issue?

**Comment by yzyboost** [comment]
21 days since this issue was posted.

**Comment by yzyboost** [comment]
Update? @paphonbth @wasdennnoch

**Comment by wasdennnoch** [comment]
NO

**Comment by Maxr1998** [comment]
@yzyboost stop asking for ETA, it's ready when it's ready. Cleaned thread.

**Comment by PradyunS** [comment]
@Maxr1998 

Device  name is galaxy s4 i9506 or ks01lte

https://download.cyanogenmod.org/?device=ks01lte

**Comment by jackbuehner** [comment]
I'm also using CyanogenMod.

HTC 10 
CM 13

Source: https://github.com/CyanogenMod/android_kernel_htc_msm8996

**Comment by yzyboost** [comment]
Pls update this issue.

**Comment by yzyboost** [comment]
pls fix this issue.

On Nov 20, 2016 10:32 PM, "paphonbth" <notifications@github.com> wrote:

From provided logs, it seems like Xposed can't find QSPanel.setTiles which
is normal on CM ROM as it has QSDragPanel instead. But the weird thing is
that if QSDragPanel exists, it should hook that class instead of QSPanel.
I can't find the cause of this yet. This is very weird.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/wasdennnoch/AndroidN-ify/issues/1221#issuecomment-261781725>,
or mute the thread

**Comment by yzyboost** [comment]
@Maxr1998

Using explay fresh. AICP Rom 6.x.x

http://4pda.ru/forum/index.php?showtopic=599456&st=14280#entry51328938

**Comment by kurtextrem** [comment]
Is you ask every two days no one will fix this. It is annoying.

**Comment by null-dev** [comment]
Some of us are watching the project so for every event that occurs, we get
an email alert. I'm one of them and it's pretty annoying when I get emails
every day containing just: "is it fixed?" or "pls update this issue".

Stop nagging the developers, nobody benefits. Notice that you have been
nagging us for quite a while and your issue is not fixed, further nagging
will not change this.

**Comment by yzyboost** [comment]
Andy, just turn off the email alert (:

Any update?

**Comment by null-dev** [comment]
I'm pretty sure I can't do it for individual issues. I either watch the
entire project, or not. And I want to watch this project because I want to
stay up to date on one of my favorite Xposed modules.

**Comment by yzyboost** [comment]
@wasdennnoch @paphonbth @Maxr1998 
[2016-12-13-00-37-10.txt](https://github.com/wasdennnoch/AndroidN-ify/files/646569/2016-12-13-00-37-10.txt)

**Comment by yzyboost** [comment]
Still no update for this issue? Wow. It's been a month since this issue was posted. When will we comply with other updates, ha? The fix huge gap is not good.

**Comment by yzyboost** [comment]
@wasdennnoch @paphonbth @Maxr1998

**Comment by yzyboost** [comment]
@wasdennnoch @paphonbth @Maxr1998 any fix for this issue? It's holiday, you have time to fix issues, please.

**Comment by yzyboost** [comment]
@wasdennnoch or are you stop building this module?

**Comment by PradyunS** [comment]
Im not even using nify anymore, too many settings and other app crashes

**Comment by yzyboost** [comment]
Pls update us? @Maxr1998 @wasdennnoch @paphonbth

**Comment by Maxr1998** [comment]
@yzyboost sadly I have to tell you again, stop asking for ETA/updates.
@wasdennnoch will definitely tell you if and when he fixes this issue.
Cleaned thread, again.
